OpenPKW/CandidateResponseDto.h

```diff
@@ -0,0 +1,21 @@
+//
+//  CandidateDto.h
+//  OpenPKW
+//
+//  Created by Piotr Tobolski on 10.05.2015.
+//  Copyright (c) 2015 OpenPKW. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CandidateResponseDto : NSObject
+
+@property (strong, nonatomic) NSString *pkwId;
```

Comment:
Wrzucanie copy wszędzie bez zastanowienia czy jest potrzebne to jest best practices? Wiem po co stosuje się copy i to nie tylko przy stringach. Ale po co to tutaj, w DTO? To nie jest libka, której może to zepsuć stan wewnętrzny czy coś.